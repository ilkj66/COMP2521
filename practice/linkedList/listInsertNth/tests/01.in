16 7 8
12
0
