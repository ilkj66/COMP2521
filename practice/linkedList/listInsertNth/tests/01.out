Enter list: Enter value to insert: Enter position to insert at: 
Original list: [16] -> [7] -> [8] -> X
List after inserting 12 at position 0: [12] -> [16] -> [7] -> [8] -> X
